modules/ROOT/pages/installation/kubernetes/helm-charts.adoc

```diff
@@ -14,57 +14,57 @@
 * Following is the reference `values.yaml` for NOM server Helm Chart:
```

Comment:
Do I understand correctly, that changes in this file reflect changes done in 15fb8968e52c75cdebe88abf8b480a2fc7d0a3ee for helm charts?

Reply:
Not yet, there is a separate for that

Reply:
Yes